.github/workflows/no-response.yml

```diff
@@ -0,0 +1,26 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # Number of days of inactivity before an Issue is closed for lack of response
+          daysUntilClose: 7
+          # Label requiring a response
+          responseRequiredLabel: needs more information
```

Comment:
Should it be "needs more information" or "status: needs more information"? I am ok with both options. cc @DanailH for X

Reply:
our current label reads `status: needs more information`. It would be great if we can keep the `status:`prefix because we have other statuses.

Reply:
I agree with the arguments, adding "status:" as a prefix helps for adding it + removing the other status labels :) Let me update.